### PR TITLE
Basic Scaffolding for AI/ML Collection

### PR DESCRIPTION
**Pull request by malsf21**
This PR targets #57, laying the groundwork for an MVP of the AI/ML collection. It does a few things:

* creates a collection `_aiml`, designed to hold a set of markdown documents for individual AI/ML lessons
* creates a layout `aiml-lesson`, which is a slide-centric page for each AI/ML lesson; creates `_aiml.scss` file with `$ai-blue` themed items
* implements `classes/aiml` page to act as a hub for the AI/ML class
* thanks to @ArjunSubramonian for adding all the new content from ACM AI Outreach's passover! also includes minor data changes
* adds override `fw` to Jekyll layouts to disable the `max-width` container in `default` layout; uses this disable in `aiml` for maximum screen real estate
* moves position of `logTopSecretMessage` in `default`, uses top-level function to log CORS issue in `aiml-lesson.html`
* removes old AI/ML lessons from resources page
* extra minor text utilities

One problem: 
* Google Slides' `iframe` embed link has a CORS issue. Seems like this is an upstream problem, not much we can do here.

Preview: https://deploy-preview-65--unruffled-perlman-fe51d2.netlify.app/classes/ml/

### CODE REVIEW

**Comment by malsf21** [comment]
@krashanoff I'm scratching my head on this CORS issue, can you take a quick peep when you have a chance? Just go to any of the lesson pages w/ an embedded slide on it.

FWIW, I'm getting the error on Chrome, quick Google didn't give me anything.

**Comment by krashanoff** [comment]
Hey Matt, thanks for getting on this. I just took a quick glance at this CORS business, and as far as I can tell, this is something that is upstream. What's going on is the iframe's embed is trying to make a CORS request but it keeps getting bounced because of restrictions on the server it's trying to hit. Something I'm willing to bet, though, is that this is by Google's design. Just a quick mockup yields the same results.

image

Odds are it'll respond when used under Google services or something to that effect. Either that or Google has a bug on their hands.

That is an annoying message, though, especially when we have our secret console messages for prospective devs. Do you know of a way of suppressing the request? I can't think of one unless we do some *very* shady JS tricks.

**Comment by malsf21** [comment]
> ...and as far as I can tell, this is something that is upstream.

Huh, that's strange. I got the embed link directly from the Google Slides > Publish To Web workflow. Do you mind taking a look and seeing if there's any other link for one of the slides? Maybe I'm being A/B tested on a new/old feature?

File: resources.html [hunk 1]
```diff
@@ -83,95 +83,17 @@ <h3 class="title">Python at Sepulveda Middle School</h3>
 <h3 class="title">Artificial Intelligence and Machine Learning at North Hollywood High School</h3>
 <ul>
         <li>
-                Lesson 1 : What is AI?
-                (
                 <a class="tla-link" target="_blank" rel="noopener noreferrer"
-                        href="https://drive.google.com/open?id=1yzc0K7pztHydPVXGULlYszm-J_OnqRxkjr5PlQ4RHDc">
-                        Slides
+                        href="/classes/ml/">
```
Comment (malsf21) [inline resources.html:87]: Note to self: add `{{site.baseurl}}` here

File: resources.html [hunk 2]
```diff
@@ -83,95 +83,17 @@ <h3 class="title">Python at Sepulveda Middle School</h3>
 <h3 class="title">Artificial Intelligence and Machine Learning at North Hollywood High School</h3>
 <ul>
         <li>
-                Lesson 1 : What is AI?
-                (
                 <a class="tla-link" target="_blank" rel="noopener noreferrer"
```
Comment (malsf21) [inline resources.html:86]: Note to self: rm `target="_blank" rel="noopener noreferrer"`

File: resources.html [hunk 3]
```diff
@@ -83,95 +83,17 @@ <h3 class="title">Python at Sepulveda Middle School</h3>
...
-                        Slides
+                        href="/classes/ml/">
+                        All Lessons
                 </a>
-                )
         </li>
         <li>
-                Lesson 2: What is ML?
-                (
                 <a class="tla-link" target="_blank" rel="noopener noreferrer"
```
Comment (malsf21) [inline resources.html:92]: Note to self: rm `target="_blank" rel="noopener noreferrer"`

File: _aiml/01-what-is-ai.md [hunk 4]
```diff
@@ -6,4 +6,4 @@ next_text: What is Deep Learning?
 slides_link: "https://docs.google.com/presentation/d/1yzc0K7pztHydPVXGULlYszm-J_OnqRxkjr5PlQ4RHDc/"
 ---
 
-You've probably heard of Artificial Intelligence and Machine Learning in the news. But what *exactly is AI & ML*?
+You've probably heard of Artificial Intelligence in the news. But what *exactly is AI*?
```
Comment (malsf21) [inline _aiml/01-what-is-ai.md:9]: Note to self: kerning issue on `?`! Include in italics

**Comment by malsf21** [comment]
Still to-do: 

* Link from `classes`
* Sticky button footers for main page
* Address review feedback

Punted to later PR: images in layout.

File: classes/aiml.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    <br><br>
+    Throughout the year, students will have numerous opportunities to learn and code with Python. Python is one of the most popular programming languages in use today, with wide-ranging applications in data processing, web development, and machine learning. Last year, our students used Python to train AI models that predict stock market prices, guess the popularity of a song on Spotify, and detect handwritten digits.
+    <br><br>
+    Learning machine learning does not occur in a vacuum! We encourage our students to think critically about the applications and ethics of AI. In the past, we’ve brainstormed which types of AI are best-suited to tackle facial recognition and estimate house prices. Furthermore, we’ve held in-depth discussions on the ethics of AI-powered self-driving cars and what it actually means for AI to be racist.
+</p>
```
Comment (malsf21) [inline classes/aiml.html:22]: Note to self: no `<br>` in final

**Comment by malsf21** [comment]
@jamieliu386 @krashanoff let me know what y'all think of this new view and setup. It's something we haven't tried before, so open to any feedback!

**Pull request merged**